Plugins~/Src/MeshSyncClientBlender/python/2.90.1/unity_mesh_sync.py

```diff
@@ -339,14 +334,14 @@ def draw(self, context):
     MESHSYNC_OT_SendAnimations,
     MESHSYNC_OT_AutoSync,
     MESHSYNC_OT_ExportCache,
-    MESHSYNC_Preferences,
-)
+    MESHSYNC_Preferences
+] + sharedClasses
 
 def register():
-    msb_initialize_properties()
     bpy.app.handlers.load_post.append(MESHSYNC_OT_AutoSync.load_handler)
     for c in classes:
         bpy.utils.register_class(c)
+    msb_initialize_properties()
```

Comment:
Properties need to be initialized after class registration because properties can use the registered classes.